docs/site/Repositories.md

```diff
@@ -9,7 +9,8 @@ summary:
 ---
 
 A Repository is a type of _Service_ that represents a collection of data within
-a DataSource.
+a DataSource. A repository class is a lightweight object that's cheap to create,
```

Comment:
`cheap` tends to have a negative connotation (at least in my head). Can we use `lightweight` or `easy`?